visualization/app/codeCharta/ui/codeMap/codeMap.component.ts

```diff
@@ -44,6 +46,10 @@ export class CodeMapController implements IsAttributeSideBarVisibleSubscriber, I
 		this.synchronizeAngularTwoWayBinding()
 	}
 
+	onClick() {
+		this.codeChartaMouseEventService.closeComponentsExceptCurrent()
```

Comment:
Do the tests cover that closeComponentsExceptCurrent() closes all except the current one?
We might should add a second unit test for it.